### PR TITLE
Czmq

### PR DESCRIPTION
**Pull request by ihsakashi**

### CODE REVIEW

File: packages/libczmq/01-support-ifaddrs.patch [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+@@ -286,9 +286,14 @@
+ #   include <sys/un.h>
+ #   include <sys/uio.h>             //  Let CZMQ build with libzmq/3.x
+ #   include <netinet/in.h>          //  Must come before arpa/inet.h
+-#   if (!defined (__UTYPE_ANDROID)) && (!defined (__UTYPE_IBMAIX)) \
+-    && (!defined (__UTYPE_HPUX))
+-#       include <ifaddrs.h>
++#   if (!defined (__UTYPE_IBMAIX)) && (!defined (__UTYPE_HPUX))
++#       if(defined(__UTYPE_ANDROID))
++#           if(defined(__ANDROID_API__)) && (__ANDROID_API__ >= 24)
```
Comment (buttaface) [inline packages/libczmq/01-support-ifaddrs.patch:14]: Just keep the original ifdef, but remove Android. No need to add all this, this repo requires API 24.

**Comment by xeffyr** [comment]
Btw, can you show examples of things requiring this library?

I can't find any request for packaging it and throwing-in a *library package* used by very few people here is not appropriate.

File: packages/libczmq/01-support-ifaddrs.patch (hunk 1 above)
Comment (ihsakashi) [reply packages/libczmq/01-support-ifaddrs.patch:14]: I will do that

**Comment by ihsakashi** [comment]
It's used in conjunction with Cap n' Proto to run https://github.com/commaai/cereal.git @xeffyr .
I understand if it's determined not appropriate.

**Comment by xeffyr** [comment]
> It's used in conjunction with Cap n' Proto to run https://github.com/commaai/cereal.git

Ok, thanks.

**Comment by xeffyr** [comment]
There is a problem with patch:
```
Applying patch: 01-support-ifaddrs.patch
patch: **** unexpected end of file in patch
```

**Comment by ihsakashi** [comment]
Sorry bout that. Thanks for fixing the patch

**Pull request merged**